addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -3816,6 +3828,34 @@ var SnippetsMenu = Widget.extend({
         // snippets could have changed on the page.
         await this._updateInvisibleDOM();
     },
+    /**
+     * Transforms an event coming from a touch screen into a mouse event.
+     *
+     * @private
+     * @param {Event} ev - a touch event
+     */
+    _onTouchEvent(ev) {
+        if (ev.touches.length > 1) {
+            // Ignore multi-touch events.
+            return;
+        }
+        const touch = ev.changedTouches[0];
+        const touchToMouse = {
+            touchstart: "mousedown",
+            touchmove: "mousemove",
+            touchend: "mouseup"
+        };
```

Comment:
I see you bind the _onTouchEvent handler to the `touchcancel` event but here there is no value for it in this `touchToMouse` object. Meaning the event handler will dispatch a simulated event with no type? :thinking:

Reply:
Indeed it was an omission on my part, I've removed it as I return as soon as there's multi-touch and the event is especially useful if the user exceeds the maximum number of touch points on their touchscreen.